admin_ui/app/modals/PermissionDeleteModal.js

```diff
@@ -25,7 +23,7 @@ const PermissionDeleteModal = () => {
     dispatch(deletePermission(permission));
   }, [dispatch, permission]);
 
-  if (!permission.id) {
+  if (displayedModal !== 'PERMISSION_DELETE' || !permission.id) {
```

Comment:
Q: Do we need this second check? When will `displayModal` be "PERMISSION_DELETE" and `permission.id` be null? Can you give an example?

Reply:
Hmmm nope we don't. This was me just coding super-defensively :-)